### PR TITLE
Add scripts to run the upgrade tests

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

File: ci/hack/plnsvc_upgrade_tests.sh [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    #     Branch, SHA or tag of the new version.
+    #     Default: Current commit.
+echo "\
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    $0 --kubeconfig mykubeconfig.yaml --from release/1.1.0 --to release/2.0.0
```
[the remark this answers is not in the document]
Comment (Roming22) [reply ci/hack/plnsvc_upgrade_tests.sh]: For the moment I've simplified the script so it uses `main` for `from` and the current branch for `to`.

We can improve it in the future to make it configurable.

[the remark this answers is not in the document]
Comment (Roming22) [reply ci/hack/plnsvc_upgrade_tests.sh]: Fixed.

File: ci/hack/plnsvc_upgrade_tests.sh [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+usage() {
+    echo "
+Usage:
+    $0 [options]
+
+Run Pipeline Service upgrade tests on the cluster referenced by KUBECONFIG.
```
Comment (gabemontero) [inline ci/hack/plnsvc_upgrade_tests.sh:13]:
```suggestion
Run Pipeline Service upgrade tests on the cluster referenced by KUBECONFIG. It uses 'main' for what you are upgrading from and the current branch for where you are upgrading to.
```

**Review by gabemontero — CHANGES_REQUESTED** [review]

File: ci/hack/plnsvc_upgrade_tests.sh (hunk 2 above)
Comment (Roming22) [reply ci/hack/plnsvc_upgrade_tests.sh:13]: Fixed.

**Review by gabemontero — APPROVED** [review]
/lgtm

**Pull request merged**